Apps/Admin/Client/Layouts/NavMenu.razor

```diff
@@ -34,7 +35,8 @@
     <AuthorizeView Roles="@Roles.Admin">
         <MudNavLink Href="support" Icon="@Icons.Material.Filled.PersonSearch">Support</MudNavLink>
     </AuthorizeView>
+    @if(ConfigurationState.Value.Result?.Features["VaccineCard"] ?? false){
```

Comment:
Braces and spacing are not following our conventions in this file.

Reply:
fixing